libs/ui/src/components/icon-button-styled/IconButtonStyled.tsx

```diff
@@ -0,0 +1,22 @@
+import styled from 'styled-components';
+import IconButton from '@mui/material/IconButton';
+import { ReactElement } from 'react';
+
+interface IconButtonStyled {
+  children: ReactElement;
+  func: () => void;
+}
+
+const _IconButtonStyled = styled(IconButton)`
+  color: #ffffff;
+`;
+
+export function IconButtonStyled({ children, func }: IconButtonStyled) {
+  return (
+    <_IconButtonStyled aria-label="logout" onClick={func}>
```

Comment:
`aria-label` sets this component to logout, so it's not really a simple icon button ? I would rename this to LogoutButton then

If this should be a generic icon button with some styling, then remove `aria-label` (or give something generic as label)